docs/website/docs/tutorial/rest-api.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Load data from a REST API
+description: How to extract data from a REST API using dlt's generic REST API source
+keywords: [tutorial, api, github, duckdb, rest api, source, pagination, authentication]
+---
+
+This tutorial shows how to extract data from a REST API using dlt's generic REST API source. The tutorial will guide you through the basics of setting up and configuring the source to load data from the API into a destination.
```

Comment:
Should we add a few lines on how easy it is to pull data from an API? Like you only need to pass the base url + some extra stuff and dlt will automatically extract everything.